payments/views.py

```diff
@@ -1,75 +1,63 @@
+import stripe
 from django.conf import settings
+from django.contrib.auth.decorators import login_required
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.core.exceptions import ObjectDoesNotExist
 from django.core.paginator import Paginator
 from django.db.models import F, Q
 from django.http import HttpResponse
-from django.shortcuts import render, redirect
+from django.shortcuts import redirect, render
+from django.urls import reverse
+from django.utils.decorators import method_decorator
 from django.views import View
 from django.views.decorators.csrf import csrf_exempt
-from django.views.generic import TemplateView
-from django.contrib.auth.decorators import login_required
-from django.utils.decorators import method_decorator
-
-from payments import forms as pay_forms
-from payments import models as pay_models
-from payments import schemas as pay_schemas
-from urllib.parse import urljoin
+from django.views.generic import FormView, TemplateView
 from users.models import Profile
-from payments.models import TopUp
-import stripe
+
+from .forms import TopUpForm
+from .models import TopUp
+from .schemas import (LineItems, LineItemsSchema, Metadata, MetadataSchema,
```

Comment:
Please use style:
```
from .schemas import (
   LineItems, 
   LineItemsSchema,
   Metadata, 
   MetadataSchema,
)
```

Reply:
Done (with a bit of a delay, but done :) )